ios/RCTConvert+BitmovinPlayer.swift

```diff
@@ -215,6 +218,67 @@ extension RCTConvert {
         return liveConfig
     }
 
+    /**
+     Utility method to instantiate a `HttpRequest` from a JS object.
+     - Parameter json: JS object.
+     - Returns: The produced `HttpRequest` object, or `nil` if `json` is not valid.
+     */
+    static func httpRequest(_ json: Any?) -> HttpRequest? {
+        guard
+            let json = json as? [String: Any?],
+            let url = RCTConvert.nsurl(json["url"]),
+            let method = json["method"] as? String,
+            let headers = json["headers"] as? [String: String]
+        else {
+            return nil
+        }
+        var request = HttpRequest(url: url, method: method)
+        request.headers = NSMutableDictionary(dictionary: headers)
+
+        if let bodyBase64EncodedString = json["body"] as? String {
+            request.body = Data(base64Encoded: bodyBase64EncodedString)
+        }
+
+        return request
+    }
+
+    /**
+     Utility method to instantiate a `HttpResponse` from a JS object.
+     - Parameter json: JS object.
+     - Returns: The produced `HttpResponse` object, or `nil` if `json` is not valid.
+     */
+    static func httpResponse(_ json: Any?) -> HttpResponse? {
+        guard
+            let json = json as? [String: Any?],
+            let request = RCTConvert.httpRequest(json["request"]),
+            let url = RCTConvert.nsurl(json["url"]),
+            let status = json["status"] as? Int,
+            let headers = json["headers"] as? [String: String]
+        else {
+            return nil
+        }
+
+        var body: Data?
+        if let bodyBase64EncodedString = json["body"] as? String {
+            body = Data(base64Encoded: bodyBase64EncodedString)
+        }
+
+        return HttpResponse(
+            request: request,
+            url: url,
+            status: status,
+            headers: headers,
+            body: body
+        )
+    }
+
+    static func networkConfig(_ json: Any?) -> NetworkConfig? {
+        guard let json = json as? [String: Any?] else {
+            return nil
+        }
+        return NetworkConfig()
+    }
```

Comment:
This func has no logic with the json because, as of now, the only things to serialise inside `NetworkConfig` are callbacks, which have to be set up in a different way and in their corresponding module.